layer.go

```diff
@@ -68,9 +67,30 @@ func (p Profile) ProcessAddf(processType string, name string, format string, a .
 	p.Addf(filepath.Join(processType, name), format, a...)
 }
 
-// Contribute represents a layer managed by the buildpack.
-type Layer struct {
+// BOMFormat indicates the format of the SBOM entry
+type BOMFormat int
+
+const (
+	CycloneDXJSON BOMFormat = iota
```

Comment:
This implies that a default of cyclonedx would be picked if not specified. Do we instead want this to be undefined and detect and error if someone tries to access that?

Reply:
I'm not sure I totally follow what you're suggestion in regards to making it undefined. Are you suggesting we add a value to the enum that is Undefined and make it the first element? I'm not very keen on that. Or is there something else I'm missing?

My present thought is that I'm fine with it being the default. Not to imply it's a better or more important format than any other format. It is just alphabetically first, so why not?